README.md

```diff
@@ -40,55 +55,70 @@ sample_rate = 44100
-    if n_iter%10==0:
-        x = vutils.make_grid(x, normalize=True, scale_each=True)
+
+    ''' scalar '''
+    dummy_s1 = torch.rand(1)
+    dummy_s2 = torch.rand(1)
+    # data grouping by `slash`
+    writer.add_scalar('data/scalar1', dummy_s1[0], n_iter)
+    writer.add_scalar('data/scalar2', dummy_s2[0], n_iter)
+
+    ''' multiple scalars '''
+    writer.add_scalars('data/scalar_group', {'xsinx': n_iter * np.sin(n_iter),
+                                             'xcosx': n_iter * np.cos(n_iter),
+                                             'arctanx': np.arctan(n_iter)}, n_iter)
+
+    dummy_img = torch.rand(32, 3, 64, 64)  # output from network
+    if n_iter % 10 == 0:
+
+        ''' image '''
+        x = vutils.make_grid(dummy_img, normalize=True, scale_each=True)
         writer.add_image('Image', x, n_iter)
-        x = torch.zeros(sample_rate*2)
+
+        ''' audio '''
+        dummy_audio = torch.zeros(sample_rate * 2)
         for i in range(x.size(0)):
-            x[i] = np.cos(freqs[n_iter//10]*np.pi*float(i)/float(sample_rate)) # sound amplitude should in [-1, 1]
-        writer.add_audio('myAudio', x, n_iter, sample_rate=sample_rate)
-        writer.add_text('Text', 'text logged at step:'+str(n_iter), n_iter)
+            # amplitude of sound should in [-1, 1]
+            dummy_audio[i] = np.cos(freqs[n_iter // 10] * np.pi * float(i) / float(sample_rate))
+        writer.add_audio('myAudio', dummy_audio, n_iter, sample_rate=sample_rate)
+
+        ''' text '''
+        writer.add_text('Text', 'text logged at step:' + str(n_iter), n_iter)
+
+        ''' histogram '''
         for name, param in resnet18.named_parameters():
             writer.add_histogram(name, param.clone().cpu().data.numpy(), n_iter)
-        writer.add_pr_curve('xoxo', np.random.randint(2, size=100), np.random.rand(100), n_iter) #needs tensorboard 0.4RC or later
+
+        ''' pr_curve '''
+        # needs tensorboard 0.4RC or later
+        writer.add_pr_curve('xoxo', np.random.randint(2, size=100), np.random.rand(100), n_iter)
+
 dataset = datasets.MNIST('mnist', train=False, download=True)
 images = dataset.test_data[:100].float()
 label = dataset.test_labels[:100]
+
+''' embedding '''
 features = images.view(100, 784)
 writer.add_embedding(features, metadata=label, label_img=images.unsqueeze(1))
 
 # export scalar data to JSON for external processing
 writer.export_scalars_to_json("./all_scalars.json")
-
 writer.close()
 ```
 
-`python demo.py`
-
-`tensorboard --logdir runs`
-
 ## Screenshots
-<img src="screenshots/Demo.gif">
 
+<img src="screenshots/Demo.gif">
 
 ## Tweaks
-To show more images in tensorboard's image tab, just
-modify the hardcoded `event_accumulator` in
-`~/anaconda3/lib/python3.6/site-packages/tensorflow/tensorboard/backend/application.py`
-as you wish.
 
-For tensorflow-tensorboard>0.17 see https://github.com/lanpa/tensorboard-pytorch/issues/44
+*built-in feature in `tensorflow-tensorboard 0.4rc`.*
```

Comment:
which feature?

Reply:
To control show how many images in Tensorboard's image tab. Does this description state this feature? Or I misunderstand :joy:

![image](https://user-images.githubusercontent.com/7029882/34607280-e0144bd2-f24d-11e7-9e6f-513a6cdec780.png)

Reply:
That should be about changing the number of visualizations per page.
As for 'show more images', I mean add more ticks for the slider. XD

Reply:
Okay, I see XD